src/prefect/cli/project.py

```diff
@@ -33,7 +34,7 @@ async def ls():
     List available recipes.
     """
 
-    recipe_paths = Path(__file__).parent / ".." / "projects" / "recipes"
+    recipe_paths = prefect.__module_path__ / "projects" / "recipes"
```

Comment:
I don't have strong feelings about this change but it's consistent with the purpose of having the module path in the init.

Reply:
Makes sense to me!